CHANGELOG.md

```diff
@@ -2,6 +2,109 @@
 
 ## Unreleased
 
+The 2.0 release contains significant breaking changes that will most likely affect all users,
+please be sure to read the migration notes below carefully.
+
+Note: 1.32.x will be security patched for at least 6 months in case some of the changes below are
+too disruptive to adopt right away.
+
+### Migration notes
+
+- Stable HTTP semantic conventions are now emitted.
+  - MANY CHANGES, see https://opentelemetry.io/blog/2023/http-conventions-declared-stable/#summary-of-changes
+- Stable JVM semantic conventions are now emitted.
```

Comment:
You think its worthwhile to make a note about attribute keys changing as well? Maybe we don't elaborate the the full set of changes, but mention that all have changed and to go read the spec for the new values..

Reply:
makes sense, I added metric attribute changes as well